<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneBtnMessageBox</class>
 <widget class="QWidget" name="OneBtnMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Com_OneBtnMessageBox_widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QLabel" name="Com_OneBtnMessageBox_lbMessage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Com_OneBtnMessageBox_bt1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
